### ISSUE TITLE
percentage font sizes don't work

### ISSUE THREAD

**Issue by rizen** [mimshwright/pixi-tagged-text#234]
Or rather, it just ignores the percent symbol and acts as "100" and "100%" are the same thing.
```
<icon fontSize="100%/> == <icon fontSize="100"/>

<tag fontSize="100%>foo</tag> == <tag fontSize="100">foo</tag>

```

**Comment by rizen** [comment]
Other sizes do the same thing.

**Comment by mimshwright** [comment]
Apparently `parseFloat("100%") === 100` 🙄 
Fixed in 3.6.1
